lib/resources/entities.js

```diff
@@ -25,6 +25,15 @@ module.exports = (service, endpoint) => {
     return Entities.getAll(dataset.id, queryOptions);
   }));
 
+  service.get('/projects/:projectId/datasets/:name/entities/creators', endpoint(async ({ Datasets, Entities }, { params, auth }) => {
+
+    const dataset = await Datasets.get(params.projectId, params.name, true).then(getOrNotFound);
+
+    await auth.canOrReject('entity.read', dataset);
```

Comment:
I feel like this should be `entity.list`. The endpoint is returning data from multiple entities. I think of `entity.read` as being about a single entity. I know we use `submission.read` for the .../submitters endpoint, but I actually think there are issues in that area: I've filed the related getodk/central#1326.